python-avd/pyavd/_eos_designs/structured_config/network_services/utils_wan.py

```diff
@@ -233,28 +237,23 @@ def _generate_wan_load_balance_policy(self: AvdStructuredConfigNetworkServicesPr
         input_dict (dict): The dictionary containing the list of path-groups and their preference.
         context_path (str): Key used for context for error messages.
         """
-        wan_load_balance_policy = {
-            "name": name,
-            "path_groups": [],
-            **get(input_dict, "constraints", default={}),
-        }
+        wan_load_balance_policy = EosCliConfigGen.RouterPathSelection.LoadBalancePoliciesItem(
+            name=name,
+            **input_topology.constraints._as_dict(),
```

Comment:
Is it possible to do without using _as_dict and `**`?

Reply:
done